ids/mapping.json

```diff
@@ -8,6 +8,7 @@
   "harshith": "harshi7016",
   "manjot": "manjots1607",
   "nikhil": "whydonti",
+  "nimish": "nimish",
```

Comment:
As per mentioned in README.md this is the expected format - ` [member_rds_id]: "<github_id>" `. Kindly replace 
"nimish": "nimish" -> "nimish" : "<your_github_id>". Thank you!

Reply:
Thanks, changing. Also, will it be better if I update this too?
```
{
	"[member_rds_id]": "[github_id]",
	"first_name": "<first-name>",
	"last_name": "<last-name>",
	"yoe": <years-of-experience>,
	"company": "<company-name>",
	"designation": "<designation>",
	"img": "./img.png",
	"github_id": "<github-id>",
	"linkedin_id": "<linkedin-id>",
	"twitter_id": "<twitter-id>",
	"instagram_id": "<instagram-id>",
	"site": "<bio-website>"
}
```

Reply:
```{
  "id": "[member_rds_id]",
  "first_name": "Ankush",
  "last_name": "Dharkar",
  "yoe": 8,
  "company": "Pesto Tech",
  "designation": "Instructor",
  "img": "URL to profile pic",
  "github_id": "ankushdharkar",
  "linkedin_id": "ankushdharkar",
  "twitter_id": "ankushdharkar",
  "instagram_id": "ankushdharkar",
  "site": "<bio-website-here>"
}```

This is the object and according to it your details are correct. So, no need of updating it. Also, can you tell from where did you find the object you mentioned above ? Thank you!

Reply:
@Rucha1499 I had written that particular object in the step 5 here: https://github.com/Real-Dev-Squad/website-static/blob/main/CONTRIBUTING.md